packages/suite/src/views/wallet/tokens/components/TokenList.tsx

```diff
@@ -1,92 +1,110 @@
+const Table = styled(Card)`
+    padding-bottom: ${spacingsPx.md};
     word-break: break-all;
 `;
 
-const TokenSymbol = styled.span`
-    ${typography.body}
-    text-transform: uppercase;
-    padding-right: 2px;
+const Columns = styled.div`
+    display: flex;
+    padding: 0 ${spacingsPx.lg};
+    border-bottom: 1px solid ${({ theme }) => theme.borderElevation2};
 `;
 
-interface ColProps {
-    $justify?: 'left' | 'right';
-    $fiatRateHidden?: boolean;
-}
+const ColName = styled.div`
+    ${typography.hint}
+    margin: ${spacingsPx.md} 0;
+    color: ${({ theme }) => theme.textSubdued};
+    width: 20%;
+`;
 
-const Col = styled.div<ColProps>`
+const Cell = styled.div<{ $isActions?: boolean; $isBigger?: boolean }>`
     ${typography.hint}
+    align-items: center;
     padding: 10px ${spacingsPx.sm} 10px 0;
-    border-top: 1px solid ${({ theme }) => theme.borderElevation2};
-
-    &:nth-child(${({ $fiatRateHidden }) => ($fiatRateHidden ? '-n + 3' : '-n + 4')}) {
-        /* first row */
-        border-top: none;
-    }
+    width: ${({ $isBigger }) => ($isBigger ? `60%` : '20%')};
+    gap: ${spacingsPx.xxs};
 
-    ${({ $justify }) =>
-        $justify &&
+    ${({ $isActions }) =>
+        $isActions &&
         css`
-            justify-content: ${$justify === 'right' ? 'flex-end' : 'flex-start'};
-            text-align: ${$justify === 'right' ? 'right' : 'left'};
+            display: flex;
+            justify-content: flex-end;
+            text-align: right;¨
+            width: 10%;
         `}
 `;
 
-const TokenName = styled.span`
-    @media only screen and (max-width: ${variables.SCREEN_SIZE.SM}) {
-        display: none;
+const Token = styled.div`
+    display: flex;
+    align-items: center;
+    padding: ${spacingsPx.xs} 0;
+    border-bottom: 1px solid ${({ theme }) => theme.borderElevation2};
+    margin: 0 ${spacingsPx.lg};
+
+    &:last-child {
+        border-bottom: none;
     }
 `;
 
-const FiatWrapper = styled.div`
-    ${typography.hint}
-    color: ${({ theme }) => theme.textSubdued};
+const TokenName = styled.span`
+    ${typography.body}
+`;
+
+const Amount = styled.div`
     display: flex;
-    justify-content: flex-end;
+    flex-direction: column;
+`;
+
+const StyledFiatValue = styled(FiatValue)`
+    ${typography.body}
+`;
+
+const StyledPriceTicker = styled(PriceTicker)`
     width: 100%;
 `;
 
 const CryptoAmount = styled(FormattedCryptoAmount)`
-    ${typography.body}
-    color: ${({ theme }) => theme.textDefault};
+    ${typography.hint}
+    color: ${({ theme }) => theme.textSubdued};
+`;
+
+const StyledTrezorLink = styled(TrezorLink)`
+    ${typography.hint}
 `;
 
-const StyledNoRatesTooltip = styled(NoRatesTooltip)`
-    justify-content: flex-end;
+const DropdownFooter = styled.div`
+    display: flex;
+    flex-direction: column;
+    border-top: 1px solid ${({ theme }) => theme.borderElevation2};
+    padding: ${spacingsPx.sm};
+    margin-top: ${spacingsPx.xxs};
```

Comment:
TokenList is extremely long component. I think DropdownFooter should have it's own component